scripts/exgdas_global_marine_analysis_post.py

```diff
@@ -59,28 +59,33 @@ def list_all_files(dir_in, dir_out, wc='*', fh_list=[]):
 post_file_list.append([os.path.join(anl_dir, 'inc.nc'),
                        os.path.join(com_ocean_analysis, f'{RUN}.t{cyc}z.ocninc.nc')])
 
-# Copy of the diagonal of the background error for the cycle
-post_file_list.append([os.path.join(anl_dir, 'static_ens', f'ocn.orig_ens_stddev.incr.{bdate}.nc'),
-                       os.path.join(com_ocean_analysis, f'{RUN}.t{cyc}z.ocn.bkgerr_stddev.nc')])
-post_file_list.append([os.path.join(anl_dir, 'static_ens', f'ice.orig_ens_stddev.incr.{bdate}.nc'),
-                       os.path.join(com_ocean_analysis, f'{RUN}.t{cyc}z.ice.bkgerr_stddev.nc')])
-
-# Copy the ice and ocean increments
-post_file_list.append([os.path.join(anl_dir, 'Data', f'ocn.3dvarfgat_pseudo.incr.{mdate}.nc'),
-                       os.path.join(com_ocean_analysis, f'{RUN}.t{cyc}z.ocn.incr.nc')])
-post_file_list.append([os.path.join(anl_dir, 'Data', f'ice.3dvarfgat_pseudo.incr.{mdate}.nc'),
-                       os.path.join(com_ocean_analysis, f'{RUN}.t{cyc}z.ice.incr.nc')])
+domains = ['ocn', 'ice']
```

Comment:
suggest not using `domains` as that makes me think `global` or `GulfOfMexico`, etc.

How about `components`?

Reply:
multi ... domain ... ufo ... 
Nope, it's staying!

Reply:
kingdom, phylum

Reply:
Ignoring @CoryMartin-NOAA 's comment!!! (bad bad G)